tests/com.espressif.idf.ui.test/src/com/espressif/idf/ui/test/executable/cases/project/NewEspressifIDFProjectTest.java

```diff
@@ -277,6 +289,32 @@ private static void turnOnDfu()
 			bot.comboBox().setSelection("DFU");
 			bot.button("OK").click();
 		}
+		
+		private static void givenESP32LaunchTargetIsSelected()
+		{
+			launchBarTargetSelector.select("esp32");
+		}
+		
+		private static void selectNewDebugLaunchConfig() {
```

Comment:
this method violates the single responsibility principle. It's ok for now, while the test is small, but it can grow really big if we decide to verify all fields in the debug configuration, it may hurt the maintainability and readability of the code. I would suggest separating this method into a few, so it's clear what we expect in tests, for example, 
replacing Fixture.selectNewDebugConfig with:
```
Fixture.whenSelectedNewDebugConfig();
Fixture.whenOpenedNewEspIdfDebuggingPage;

Fixture.thenProjectFieldIsTheSameAsSelectedProject;
Fixture.thenActualExecutableIsCorrect;

Fixture.finishNewEspIdfDebuggingConfiguration;
```